docker/dependencies/Dockerfile

```
@@ -7,6 +7,7 @@ ARG CMAKE_BUILD_TYPE=Release
 
 ENV IROHA_HOME /opt/iroha
 ENV IROHA_BUILD /opt/iroha/build
+ENV PATH /root/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

You can just prepend a path to whatever already exists:
```suggestion
ENV PATH "/root/.cargo/bin:${PATH}"
```